tools/src/main/java/org/bitcoinj/tools/WatchMempool.java

```diff
@@ -42,7 +42,7 @@ public class WatchMempool {
     private static final Map<String, Integer> counters = new HashMap<>();
     private static final String TOTAL_KEY = "TOTAL";
     private static final long START_MS = System.currentTimeMillis();
-    private static final long STATISTICS_FREQUENCY_MS = 1000 * 5;
+    private static final long STATISTICS_FREQUENCY_MS = 5_000;
```

Comment:
This could be converted to `java.time` API instead. Here is a PR: https://github.com/bitcoinj/bitcoinj/pull/2830